### DISCUSSION TITLE
Hook for when all prendering/generate is done.

### DISCUSSION THREAD

**Discussion by surgiie** [nitrojs/nitro#1399] [Q&A] [answered]
Im looking for a hook that occurs after ALL routes have been processed/generated but cant seem to find any documentation or indication that its supported from looking at the code here https://github.com/unjs/nitro/blob/main/src/prerender.ts#L40

Am i missing something or is there no support for this?

**Comment by Hebilicious** [comment, marked as answer]
> Im looking for a hook that occurs after ALL routes have been processed/generated but cant seem to find any documentation or indication that its supported from looking at the code here [`main`/src/prerender.ts#L40](https://github.com/unjs/nitro/blob/main/src/prerender.ts#L40)
> 
> Am i missing something or is there no support for this?

If you're using Nuxt, you can have a look at the nuxt devtools to see if one of the existing hook would fit your use-case https://nuxt.com/docs/api/advanced/hooks based on when they run.
